.travis.yml

```diff
@@ -31,6 +31,7 @@ env:
   global:
     - ES7_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.3.0-linux-x86_64.tar.gz"
     - ES_HOST=127.0.0.1
+    - PIPENV_PYUP_API_KEY=""
```

Comment:
is this needed?

Reply:
I was getting `Your API Key is invalid` error, https://github.com/pypa/pipenv/issues/4188 error, which was also the case for multiple invenio repositories during the last sprint.